src/trunk/libs/seiscomp3/datamodel/utils.h

```diff
@@ -65,7 +65,7 @@ char objectStatusToChar(const T *o) {
 	catch ( ... ) {}
 
 	try {
-		if ( o->evaluationMode() == 'M' )
+		if ( o->evaluationMode() == DataModel::MANUAL )
```

Comment:
SInce we are already in the namespace DataModel, please remove it and just use ```MANUAL```.